rocky/reports/report_types/mail_report/report.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from datetime import datetime
+from logging import getLogger
+from typing import Any, Dict
+
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models import Reference
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6
+from reports.report_types.definitions import Report
+
+logger = getLogger(__name__)
+
+
+@dataclass
+class System:
+    system_types: list
+    oois: list
```

Comment:
Could you provide more specificity about the content of the list? If uncertain, consider using `Any` from the `typing` module (e.g., `List[Any]`).

Also, for Python 3.8 we use `typing.List` (like the previous example) instead of `list` for annotating list types